src/lib.rs

```diff
@@ -24,7 +24,7 @@ pub mod metrics;
 pub mod server;
 pub mod storage;
 
-pub use crate::server::Server;
+pub use crate::server::ftpserver::Server;
```

Comment:
Clippy doesn't like a repeat of the module name e.g. `server::server` so I made it `server::ftpserver`. Feel free to suggest better names.